src/pybind/mgr/orchestrator/module.py

```diff
@@ -821,8 +822,23 @@ def _list_daemons(self,
 
         daemons = raise_if_exception(completion)
 
+        valid_fields = ['name', 'host', 'image']
```

Comment:
create an Enum of these and add them to the cli args so the valid options will display via `--help`? 

similar to:
https://github.com/ceph/ceph/blob/5cd4a0ec53af46e2b43e18d9ecadd8524f24baa5/src/pybind/mgr/orchestrator/module.py#L1214

Reply:
Thanks, a very nice feature of the arg parser indeed, I didn't know about :)